### PR TITLE
Reduced Performance and Productivity sections.

### PR DESCRIPTION
**Pull request by mjawili**
Consolidated all into a single slide (and one title slide).

### CODE REVIEW

File: csstools/index.html [hunk 1]
```diff
@@ -786,7 +786,34 @@
...
+      <section class="content"> 
+        <p>Performance</p>
+        <ul>
+          <li>CSS file size and compression
+          <li>Image sprites
+          <li>Data URI
+          <li>Icon fonts
+        </ul>
+        <p>CSS preprocessors</p>
+        <ul>
```
Comment (pamelafox) [inline csstools/index.html]: I kind of think they won't understand what preprocessors are without the example. How about:
CSS workflow in large apps:
- Linters (CSSLint)
- Minifiers (YUI, CSMinifier)
- Preprocessors (Less, Sass)
- Build tools (Makefile, Grunt, Gulp)
with links to each.
Basically, they don't need to care about all those things unless they're making a webapp of a significant size, so I think that bucketing makes sense.

File: csstools/index.html [hunk 2]
```diff
@@ -786,7 +786,34 @@
...
+    
+    <div class="slide normal"> 
+      <header><h1>Other things worth considering</h1></header> 
+      <section class="content"> 
+        <p>Performance</p>
+        <ul>
+          <li>CSS file size and compression
+          <li>Image sprites
+          <li>Data URI
+          <li>Icon fonts
```
Comment (pamelafox) [inline csstools/index.html]: Maybe:
Improving rendering speed with CSS:
- Image sprites (with a link to an article)
- Data URIs ( with a link to an article)
- Icon fonts (with a link to an article)
Probably CSStricks articles or whatever is top Google result

**Comment by pamelafox** [comment]
Looks great!

**Comment by mjawili** [comment]
Awesome. I'll submit more throughout the day...

**Pull request merged**